livekit-plugins/livekit-plugins-assemblyai/livekit/plugins/assemblyai/stt.py

```diff
@@ -289,6 +289,8 @@ async def recv_task(ws: aiohttp.ClientWebSocketResponse):
                 except Exception:
                     logger.exception("failed to process AssemblyAI message")
 
+        ws: aiohttp.ClientWebSocketResponse | None = None
```

Comment:
We didn't add assemblyai to the CI?

Reply:
we have, but test case was failing because it wasn't able to access `ws` in `finally` https://github.com/livekit/agents/blob/main/livekit-plugins/livekit-plugins-assemblyai/livekit/plugins/assemblyai/stt.py#L313